code-challenges.test.js

```diff
@@ -17,78 +17,173 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var num1 = 15
-// Expected output: "15 is divisible by three"
-
-var num2 = 0
-// Expected output: "0 is divisible by three"
-
-var num3 = -7
-// Expected output: "-7 is not divisble by three"
-
-
-
-
+// Name of the function will be 'divisibleByThree'
+describe("divisbleByThree", () => {
+    var num1 = 15
+    // Expected output: "15 is divisible by three"
+    it("return '15 is divisible by three' for an input of number 15", () => {
+        expect(divisibleByThree(num1)).toEqual('15 is divisible by three')
+    })
+
+    var num2 = 0
+    // Expected output: "0 is divisible by three"
+    it("return '0 is divisible by three' for an input of number 0", () => {
+        expect(divisibleByThree(num2)).toEqual('0 is divisible by three')
+    })
+
+    var num3 = -7
+    // Expected output: "-7 is not divisble by three"
+    it("return '-7 is not divisible by three' for an input of number -7", () => {
+        expect(divisibleByThree(num3)).toEqual('-7 is not divisible by three')
+    })
+})
 
 // b) Create the function that makes the test pass.
 
-
+// Create a function that takes in 1 argument that is a number
+const divisibleByThree = (number) => {
+    // The number will check if it is divisible by three by using the modulo
+    // and check that the remainder is zero
+    // Case 1 - it IS divisible by 3
+    if (number % 3 === 0) {
+        // Return a string saying the number is divisible by 3
+        return `${number} is divisible by three`;
+    // Case 2 - it is NOT divisible by 3
+    } else if (number % 3 !== 0) {
```

Comment:
because the `else if` conditional is the exact negation of the one in the `if` statement, it's fair to use `else` block instead of `else if`.